src/clj/ataru/applications/excel_export.clj

```diff
@@ -131,18 +131,38 @@
             (str filename " (" (util/size-bytes->str size) ")")))
         value))))
 
+(defn- sec-or-vec? [value]
```

Comment:
I think `sequential?` would be equivalent?

Reply:
:+1: